File: src/Learnify/Learnify.Core/Dto/Course/QuizQuestion/QuizQuestionUpdateResponse.cs
using Learnify.Core.Dto.Course.QuizQuestion.Answers;

namespace Learnify.Core.Dto.Course.QuizQuestion;

public class QuizQuestionUpdateResponse
{
    public string Id { get; set; }
    public string Question { get; set; }
    public AnswersUpdateResponse Answers { get; set; }
}

File: front/client/src/app/Models/Course/Lesson/QuizQuestion/QuizQuestionUpdateResponse.ts
import { AttachmentResponse } from "src/app/Models/Attachment/AttachmentResponse";
import { AnswersUpdateResponse } from "./Anwers/AnswersUpdateResponse";
import { CurrentLessonUpdatedResponse } from "../CurrentLessonUpdatedResponse";

export interface QuizQuestionUpdateResponse {
    id: string;
    question: string;
    answers: AnswersUpdateResponse;
}